### ISSUE TITLE
Provide a better user experience on the Quickstart docs page

### ISSUE THREAD

**Issue by mikenikles** [gitpod-io/website#922]
Many new quickstart guides have been added to https://www.gitpod.io/docs/quickstart recently. Both the navigation and the Quickstart landing page have therefore become a bit long.

I suggest the following
- Remove all individual quickstart menu items from the nav
- Display quickstarts at https://www.gitpod.io/docs/quickstart in a grid, sorted alphabetically with a search field at the top that filters the results as someone types

**Comment by JohannesLandgraf** [comment]
>Remove all individual quickstart menu items from the nav

+1

>Display quickstarts at https://www.gitpod.io/docs/quickstart in a grid, sorted alphabetically with a search field at the top that filters the results as someone types

I like the idea of the search field. But shouldn't we have the top 5 (most clicked) permanent and after that start some sorting? Also adding references from Vercel [[1]](https://vercel.com/new/templates)[[2]](https://vercel.com/docs#quickstarts)

**Comment by mikenikles** [comment]
Yep, let's do that. The search will be below the 5 permanent quickstarts.

**Comment by Cahllagerfeld** [comment]
I'd like to work on that if noone already started with it 😄 
Is there currently a mechanism on detecting the 'most-used'-quickstart?
